### DISCUSSION TITLE
可以多分区么

### DISCUSSION THREAD

**Discussion by Hby118** [apache/doris#21502] [Ideas]
比如多个LIST 分区
或者像多字段range分区一样 多字段list分区
或者LIST + RANGE

手册上没查到相关方法,介绍的都是单分区
自己尝试又总是报错

**Comment by Hby118** [comment]
List分区也支持多列分区，示例如下：

PARTITION BY LIST(`id`, `city`)
(
    PARTITION `p1_city` VALUES IN (("1", "Beijing"), ("1", "Shanghai")),
    PARTITION `p2_city` VALUES IN (("2", "Beijing"), ("2", "Shanghai")),
    PARTITION `p3_city` VALUES IN (("3", "Beijing"), ("3", "Shanghai"))
)

手册上看到了

是不是如果要多列只能这样拼起来,不能多列字段都独立一个分区?

**Reply by yagagagaga** [reply]
是的

**Reply by Hby118** [reply]
谢谢
